bigflow/scaffold/templates/new-project/resources/requirements.txt

```diff
@@ -0,0 +1,81 @@
+# *** AUTO GENERATED: DON'T EDIT ***
+# $source-hash: sha256:536b5d9967c9983def6b9ee11127963a5a209d1713d6c4a90114c0789fb0c373
+# $source-file: resources/requirements.in
+#
+# Run 'bigflow pip-compile resources/requirements.in' to update this file
```

Comment:
`pip-compile` is not very meaningful if you don't know the tool. Maybe `build-requirements` (because it's a part of the build process)?

Reply:
Good idea. Renamed it to 'build-requirements'.